schema-generator-plugin/src/main/java/com/github/jrcodeza/schema/generator/plugin/GenerateOpenApiSchemaMojo.java

```diff
@@ -104,6 +119,14 @@ public void execute() {
 		}
 	}
 
+	private <T> T parseInputFilter(String className, Class<T> clazz) {
+		if (className == null || className.isEmpty()) {
```

Comment:
You can use StringUtils.isNotEmpty(className) it does the null check and empty check automatically

Reply:
ok I changed that